app/src/main/java/org/oppia/android/app/topic/lessons/TopicLessonsFragment.kt

```diff
@@ -145,4 +145,14 @@ class TopicLessonsFragment :
   ) {
     topicLessonsFragmentPresenter.selectChapterSummary(storyId, explorationId, chapterPlayState)
   }
+
+  /** Returns the [currentExpandedChapterListIndex] stored in the fragment. */
+  fun getCurrentExpandedChapterListIndex(): Int? {
+    return currentExpandedChapterListIndex
+  }
+
+  /** Returns the [isDefaultStoryExpanded] stored in the fragment. */
+  fun getIsDefaultStoryExpanded(): Boolean {
+    return isDefaultStoryExpanded
+  }
```

Comment:
Rather than exposing internal state variables for test verification, is it possible to verify that the UI is in the correct state such that we can infer these? For example, could we check that the (expected) default story is currently expanded?